### ISSUE TITLE
Extreme Reactors Ore - Config Change request

### ISSUE THREAD

**Issue by jeffwcollins** [AllTheMods/ATM-3#58]
Noticed that Extreme Reactors ore is setup to generate in small clusters, and only white listed for the overworld.  I request whitelisting the ore in the other worlds, such as the deep dark, Aroma mining dimension, and the ExUtils Mining dimension.  Also wouldn't hurt if you bumped up the cluster size by just a hair due to the lack of higher power production methods in this pack, requiring more use of yellorium.

**Comment by OrdinatorStouff** [comment]
I agree. Relevant config is in **Extreme Reactors.cfg**:
```
    I:dimensionWhitelist <
        0
        6
        -11325
     >
```

As for cluster size/etc idk -- reactors can be very "OP" -- maybe leave as defaults since 1) it'll be more accessible in mining dimensions and 2) **IF** mystical agriculture is added, yellorium seeds can be grown

**Comment by jeffwcollins** [comment]
The exutil dimension is -9999.

An increase in cluster size can be offset by using the general configs that the FTB Team uses for fuel usage.

**Comment by OrdinatorStouff** [comment]
-9999 is the quantum quarry dim which yeah we can add that too except Exu2 is dangerouslly buggy and laggy we've seen. If Exu2 stays in the pack in the first place, that quarry and it's dim might be disabled indefinitely. Still makes sense to include it in the configs nonetheless.

**Comment by whatthedrunk** [comment]
fixed in version 2.2 that will be out soon
